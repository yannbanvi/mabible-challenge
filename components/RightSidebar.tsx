import { Flex } from "@chakra-ui/react"

function RightSidebar() {
  return (
    <Flex 
        data-testid='right-sidebar' 
        borderLeft='2px solid #F1F1F2' 
        w='425px' 
        h='100%' 
        direction='column'
        backgroundColor='#FAFAFB'>

    </Flex>
  )
}

export default RightSidebar