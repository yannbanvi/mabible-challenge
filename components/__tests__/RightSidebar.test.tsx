/** @jest-environment jsdom */
import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import RightSidebar from '../RightSidebar'
describe('RightSidebar', () => {

  it('should render RightSidebar successfully', () => {
    const { container } = render(<RightSidebar />);
    expect(container).toMatchSnapshot();
  });
});