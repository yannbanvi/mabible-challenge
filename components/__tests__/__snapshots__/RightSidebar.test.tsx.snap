// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RightSidebar should render RightSidebar successfully 1`] = `
<div>
  <div
    class="css-9pknlg"
    data-testid="right-sidebar"
  />
</div>
`;
